napari/layers/shapes/_shapes_models/shape.py

```diff
@@ -149,10 +155,18 @@ def __init__(
 
     def __new__(cls, *args, **kwargs):
         if (
-            get_settings().experimental.compiled_triangulation
-            and triangulate_path_edge_numpy is not None
+            TriangulationBackend.bermuda == TRIANGULATION_BACKEND
+            and bermuda is not None
+        ):
+            cls._set_meshes = cls._set_meshes_compiled_bermuda
+        elif TriangulationBackend.partsegcore == TRIANGULATION_BACKEND:
+            assert partsegcore_triangulate is not None
```

Comment:
why does this have an assert statement but triangle and bermuda don't?

Reply:
ops. Leftover after debug of benchamarks